2_observations/src/crosswalk_functions.R

```diff
@@ -0,0 +1,40 @@
+
+crosswalk_sites_to_reaches <- function(network_ind, boundary_ind, sites_ind, out_ind) {
+  
+  # read in network and boundary files
+  drb_net <- readRDS(sc_retrieve(network_ind))
+  drb_boundary <- readRDS(sc_retrieve(boundary_ind))
+  
+  # read in sites from WQP, nwisdv, nwisuv
+  sites <- readRDS(sc_retrieve(sites_ind)) %>%
+    filter(site_type %in% c('ST', 'Stream'))
+  
+  # Convert to sfc
+  obs_site_points <- purrr::map2(sites$longitude, sites$latitude, function(lat, lon) {
+    st_point(c(lat, lon), dim='XY')})
+  
+  obs_sites <- sites %>%
+    st_set_geometry(st_sfc(obs_site_points)) %>%
+    st_set_crs(4326) %>%
+    st_transform(crs=st_crs(drb_net$vertices))
+  
+  # Subset to sites in the Delaware River Basin
+  drb_sites <- obs_sites[st_intersects(drb_boundary, obs_sites)[[1]], ] # 5009 rows
+  
+  #### Match sites to reaches ####
+  
+  source('2_observations/src/subset_closest.R')
```

Comment:
is this `source` call related to https://github.com/USGS-R/delaware-model-prep/pull/2/files#diff-92840c0a5bf8c385cefec33dd1bdb072R35?

Reply:
I'm wondering if this `depends` and `source` pattern is just a remnant of my one-off approach to drafting this code - I see that I have the same call in my code in `delaware-prms`. I think/hope it shouldn't be necessary now that you've scipiperized. (scipiped?)

Reply:
[scipipified?]

See above comments about what I thought was going on.